modules/apps/27-interchain-accounts/controller/keeper/msg_server.go

```diff
@@ -62,7 +62,8 @@ func (s msgServer) SendTx(goCtx context.Context, msg *types.MsgSendTx) (*types.M
 		return nil, sdkerrors.Wrap(channeltypes.ErrChannelCapabilityNotFound, "module does not own channel capability")
 	}
 
-	seq, err := s.Keeper.SendTx(ctx, chanCap, msg.ConnectionId, portID, msg.PacketData, msg.TimeoutTimestamp)
+	absoluteTimeout := uint64(ctx.BlockTime().UnixNano()) + msg.TimeoutTimestamp
+	seq, err := s.sendTx(ctx, chanCap, msg.ConnectionId, portID, msg.PacketData, absoluteTimeout)
```

Comment:
I'm not 100% clear if there is more required than this change with the private function.